shifter-support-plugin.php

```diff
@@ -10,21 +10,63 @@
 License: GPL2
 */
 
+
+
+/*
+ * Shifter API
+ */
+
 require("api/shifter_api.php");
 
-define('SHIFTER_ASSET_DIR', content_url('/mu-plugins/') . basename(__DIR__) . '/assets/');
-define('SHIFTER_CSS', SHIFTER_ASSET_DIR . 'css/shifter-support.css');
-define('SWAL_JS', SHIFTER_ASSET_DIR . 'js/sweetalert.min.js');
 
-add_action("admin_enqueue_scripts", "add_shifter_support_css");
+
+/*
+ * Assets
+ * Dirs and Vars
+ */
+
+define('SHIFTER_ASSET_DIR', content_url('/mu-plugins/') . basename(__DIR__) . '/src/');
```

Comment:
It seems we have to replace to `/dist/` when release. could you get path dynamically from current environment ?